src/views/sign-in/external/KB_SSO/template/KB_SSO.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="kbSSO-wrapper">
+        <form id="myForm" @submit.prevent="openKBSSO">
+            <p-button
+                style-type="primary1 outline"
+                size="lg"
+                class="kbSSO-btn"
+            >
+                KB SSO
+            </p-button>
+        </form>
+    </div>
+</template>
+
+<script lang="ts">
+import axios from 'axios';
+import {
+    ComponentRenderProxy, defineComponent, getCurrentInstance,
+} from '@vue/composition-api';
+import { PButton } from '@spaceone/design-system';
+import { loadAuth } from '@/views/sign-in/authenticator/loader';
+
+export default defineComponent({
+    name: 'KBSignIn',
+    components: {
+        PButton,
+    },
+    setup(props, { context, emit }) {
```

Comment:
이 부분에서 타입에러가 납니다.
context 는 SetupContext에 없습니다.
```suggestion
    setup(props, { emit }) {
```